doc/try/index.rst

```diff
@@ -6,16 +6,34 @@ Try using ACRN
 After reading the :ref:`introduction`, use these guides to get started
 using ACRN in a reference setup.  We'll show how to set up your
 development and target hardware, and then how to boot up the ACRN
-hypervisor and the `Clear Linux`_ Service OS and Guest OS on the Intel
+hypervisor and the `Clear Linux`_ Service VM and User VM on the Intel
 (EFI) platform.
 
 .. _Clear Linux: https://clearlinux.org
 
 ACRN development is currently supported on Apollo Lake Intel platforms,
-described in :ref:`hardware`. Follow the setup guides listed here:
+described in :ref:`hardware`.
+
+For **SDC** scenario user, follow the setup guides listed here:
 
 .. toctree::
    :maxdepth: 1
 
    apl-nuc.rst
    up2.rst
+
+For **SDC2** scenario user, follow the setup guides listed here:
```

Comment:
We can't add references to documents that don't exist yet (it will break the doc build)